spec/index.bs

```diff
@@ -1324,7 +1330,98 @@ dictionary GPUBufferBinding {
 };
 </script>
 
-  * {{GPUBufferBinding/size}}: If undefined, specifies the range starting at {{GPUBufferBinding/offset}} and ending at the end of the buffer.
+  * {{GPUBufferBinding/size}}: If undefined, specifies the range starting at
+      {{GPUBufferBinding/offset}} and ending at the end of the buffer.
+
+A {{GPUBindGroup}} object has the following internal slots:
+
+<dl dfn-type=attribute dfn-for="GPUBindGroup">
+    : <dfn>\[[layout]]</dfn> of type {{GPUBindGroupLayout}}.
+    ::
+        The {{GPUBindGroupLayout}} associated with this {{GPUBindGroup}}.
+
+    : <dfn>\[[bindings]]</dfn> of type sequence<{{GPUBindGroupBinding}}>.
+    ::
+        The set of {{GPUBindGroupBinding}}s this {{GPUBindGroup}} describes.
+</dl>
+
+### {{GPUDevice/createBindGroup()|GPUDevice.createBindGroup(GPUBindGroupDescriptor)}} ### {#GPUDevice-createBindGroup}
+
+<div algorithm="GPUDevice.createBindGroup(descriptor)">
+
+The <dfn method for="GPUDevice">createBindGroup(|descriptor|)</dfn> method is used to create
+    {{GPUBindGroup}}s.
+
+1. Ensure [=bind group device validation=] is not violated.
+1. Ensure |descriptor|.{{GPUBindGroupDescriptor/layout}} is a valid {{GPUBindGroupLayout}}.
```

Comment:
```suggestion
1. Ensure |descriptor|.{{GPUBindGroupDescriptor/layout}} is a [=invalid|valid=] {{GPUBindGroupLayout}}.
```

Or maybe that's confusing.

Instead, we could add a definition for `<dfn dfn>valid</dfn>`, or perhaps `<dfn dfn>is a valid</dfn>`, up near the definition of `invalid`. Up to you.

Reply:
Done. Add a simple sentence for "valid" definition.